src/ratgdo.cpp

```diff
@@ -155,17 +174,19 @@ void setup_pins()
 
     pinMode(INPUT_OBST_PIN, INPUT);
 
-    /*
-     * TODO add support for dry contact switches
+   
     pinMode(STATUS_DOOR_PIN, OUTPUT);
-    */
+    
     pinMode(STATUS_OBST_PIN, OUTPUT);
-    /*
+    
     pinMode(DRY_CONTACT_OPEN_PIN, INPUT_PULLUP);
     pinMode(DRY_CONTACT_CLOSE_PIN, INPUT_PULLUP);
-    pinMode(DRY_CONTACT_LIGHT_PIN, INPUT_PULLUP);
```

Comment:
Are we not able to control the light with dry contact mode?

Reply:
no, light light controls are analog on most dry contact openers.

Reply:
so, @PaulWieland , it would make sense to remove the light service from HomeKit so that a switch that can't do anything is not shown in Apple Home.  Correct?  Only for dry-contact mode of course.

Reply:
Yes agreed. With dry contact control the only thing you can do is open and close the door (and get door status feedback)